lib/modules/dosomething/dosomething_api/resources/kudos_resource.inc

```diff
@@ -189,6 +188,29 @@ function _kudos_resource_index($reportbackitems, $count) {
  * @param $term_ids
  */
 function _kudos_resource_create($reportback_item_id, $term_ids, $northstar_id) {
+  if (variable_get('rogue_collection', FALSE)) {
+    global $user;
+    $northstar_user = dosomething_northstar_get_user($user->uid, 'drupal_id');
+
+    $data = [
+      'northstar_id' => $northstar_user->id,
+      'post_id' => $reportback_item_id,
+    ];
+
+    $response = dosomething_rogue_client()->postReaction($data);
```

Comment:
I think it's okay letting this 500 if there are any problems – we can always revisit and add better error handling then.